### ISSUE TITLE
Required rule does not work

### ISSUE THREAD

**Issue by SigalZ** [proengsoft/laravel-jsvalidation#797]
### Subject of the issue
The required rule displays an error only if you type something in the required field and then delete what you typed.
If you just skip this field by pressing Tab, no error is displayed.

### Your environment
* version of this package: 4.6.1
* version of Laravel: 8

### Steps to reproduce
Create this validator in a controller:
```
protected $validationRules = [
        'weight' => 'numeric|required',
        'colour' => 'required',
        'stock' => 'numeric',
        'low_stock_warn' => 'numeric',
    ];

public function create()
{
        $validator = JsValidator::make($this->validationRules);
        
        return view('admin.bags.create')->with(['validator' => $validator]);
}

public function store(Request $request)
    {
        $validation = Validator::make($request->all(), $this->validationRules);
    
        if ($validation->fails()) {
            return redirect()->back()->withErrors($validation->errors());
        }

        $active = empty($request['active']) ? 0 : 1;
		
        $bag = Bag::create([
            'weight' => $request->weight,
            'colour' => $request->colour,
            'stock' => $request->stock,
            'low_stock_warn' => $request->low_stock_warn,
            'active' => $active
        ]);

        $bag->save();
        
        alert()->success('Congrats!', 'You created a bag');

        return Redirect::route('bags.index');
    }
```
Create a form with these fields and of course add all the necessary js etc.

Run the form in the browser, click on the weight field then click 'Tab' to move to the next field, no error displays to say the weight field is required.

Click 'Tab' to move on from the colour field, again, no error is displayed.

Type something in the colour field, click 'Tab' then delete what you typed in the colour field, the required error appears.

### Expected behaviour
If a field is required, I expect to see an error when I leave it blank and move to the next field.

### Actual behaviour
When I leave the field empty and move to the next field, no error is displayed.

**Comment by SigalZ** [comment]
I just found out, that it works only if I use backspace to delete what I captured. If I use the 'delete' button to empty a field, again I don't get the required error.

**Comment by bytestream** [comment]
This is an issue for upstream [`jquery-validation/jquery-validation`](https://github.com/jquery-validation/jquery-validation/), or the browser that you're using. `TAB` should cause a `focusin` and `focusout` event which should cause validation to occur - https://github.com/jquery-validation/jquery-validation/blob/31ea8ffc4de5547c828001ff3ba95f72e2648617/src/core.js#L427

Failing that, validation will always happen when `submit` is pressed
